src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Colors[] colors = Colors.values();
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = this.random.nextInt(this.colors.length);
+        return this.colors[index].name();
```

Comment:
read about when to use this keyword. otherwise better not to use it, it makes no sense
```suggestion
        int index = random.nextInt(colors.length);
        return colors[index].name();
```